web/client/epics/dashboard.js

```diff
@@ -180,6 +183,28 @@ export const saveDashboard = action$ => action$
             )
     );
 
+export const exportDashboard = action$ => action$
+    .ofType(DASHBOARD_EXPORT)
+    .switchMap(({originalData, resource, services}) =>
+        Rx.Observable.of([JSON.stringify({originalData, resource, services}), 'dashboard.json', 'application/json'])
```

Comment:
I would remove the resource infomation because they are related to the dashboard seved in the geostore, I think this is meant as a standalone file. Maybe we could use the resource name to generate the name of  the json.
Proposals:
- remove the resource key from the export dashboard
- generate the name of the exported json based on the name of resource
```js
`${resource?.name ? ${resource.name}-` : ''}dashboard.json`
```